examples/vm-priorityclass.yaml

```diff
@@ -0,0 +1,41 @@
+---
+apiVersion: kubevirt.io/v1alpha3
+kind: VirtualMachine
+metadata:
+  labels:
+    kubevirt.io/vm: vm-cirros
+  name: vm-non-preemtible
+spec:
+  running: false
+  template:
+    metadata:
+      labels:
+        kubevirt.io/vm: vm-cirros
+    spec:
+    // deploy priorityClass first
```

Comment:
i don't think `//` is a valid way to comment in yaml. maybe I'm wrong though? i thought `#` was the way to do it.

Reply:
you are right... will be removed as this will be autogenerated